modules/Stories/LoadMore.tsx

```diff
@@ -0,0 +1,26 @@
+import type { FunctionComponent } from 'react';
+
+type Props = {
+    canLoadMore: boolean;
+    isLoading: boolean;
+    onLoadMore: () => void;
+};
+
+const LoadMore: FunctionComponent<Props> = ({ canLoadMore, isLoading, onLoadMore }) => {
+    if (!canLoadMore) {
```

Comment:
Hmm, very minor, but do you think it is better to put this condition here rather than in a parent
([InfiniteStories.tsx :28](https://github.com/prezly/theme-nextjs-starter/pull/14/files#diff-0f613475865e97ed1b14883f98383deb2135685c2e1386542dab985eeb4cba29R28))?

Reply:
I don't have strong preference here, but probably putting it in parent component is a bit more clear.